dhis-2/dhis-api/src/main/java/org/hisp/dhis/programstagefilter/EventDataFilter.java

```diff
@@ -65,6 +65,12 @@ public class EventDataFilter implements Serializable {
   /** Like */
   private String like;
 
+  /** Null */
+  private String isNull;
```

Comment:
to be consistent with the other attributes, simply having "null" as the property name would be better right?
Also for the opposite "notNull" maybe. 🤔

Reply:
I agree, but "null" is a keyword in java, I could rename it to `nullValue` or `nullFilter`, or something else that might make more sense.